### ISSUE TITLE
Publish to Maven Central

### ISSUE THREAD

**Issue by cowwoc** [robinbraemer/CloudflareAPI#50]
Obvious question: Can you please publish your library to Maven Central? It would make it a lot easier for end-users to work with it.

**Comment by robinbraemer** [comment]
Thanks for the feedback.

It currently uses Jitpack, is there any unusability issue with that?

**Comment by xgp** [comment]
Two issues:
1. The way maven works, when you add a repository, it checks all repositories for all libraries, and you can't just limit it by the library.
2. Related to the above, many organizations have policies that require them to use only certain "vetted" repositories.
3. Maven Central is considered the "standard", and many people will not use something that isn't published there.

Let me know if you'd accept a PR, and I can add a github action and make the pom changes that will auto publish for you.

**Comment by robinbraemer** [comment]
Sure, I'm looking forward to your PR. It would be helpful if you could provide some steps along with it. (any registration needed?)

**Comment by xgp** [comment]
Yes. You'll need to create an account on Sonatype, and validate your "ownership" of your `groupId`, but I will send links to those steps in the PR. 

This is an example of the github action definition that will do it https://github.com/p2-inc/keycloak-orgs/blob/main/.github/workflows/release.yml
It uses this action https://github.com/qcastel/github-actions-maven-release
